### ISSUE TITLE
Dashboards - Being able to create a trend line rather than all "wobbly" datapoints

### ISSUE THREAD

**Issue by Angelodaniel** [getsentry/sentry#77070]
### Problem Statement

In some widgets with a lot of ups and downs it will make a very chaotic view where it's hard to spot if a certain release had any impact on the results.

![Image](https://github.com/user-attachments/assets/a42301a3-1986-46e2-9998-0ac20f898f86)

### Solution Brainstorm

It would be nice to "smoothen" out the lines
So that it's not so "wobbly" and maybe easier to spot overal ups and downs?
More of like a trend line.

### Product Area

Dashboards

**Comment by gggritso** [comment]
This is an interesting problem area. From Slack:

> there’s definitely something here! When I look at graphs I also often have questions like “is this line going up?” or “what’s the cycle time of this sine wave” or “what’s the time duration of this spike
>
> there are all kinds of possible interrelated solutions here. Horizontal cursors, cursor locking, trendlines, smoothing, measurement tools, who knows

**Comment by Angelodaniel** [comment]
Hey Team, I guess being able to select the timeframe like we do for alerts or discover would already help @gggritso 
Any update on this work?

**Comment by gggritso** [comment]
@Angelodaniel no update that I know of, sorry! By "timeframe" do you mean selecting the interval (e.g., 15m, 10m, etc.)? @benjamin.coe would know more if this work is planned

**Comment by Angelodaniel** [comment]
@gggritso yes exactly, if I can "smoothen" out the line by determining how spiky the line is based on the timeframe that would fix it + I think we already have this behaviour in some of our graphs throughout the platform

**Comment by bcoe** [comment]
No updates on this work. In the short term, we will be prioritizing making insights work on the dashboards platform, with relatively few new features added to dashboards themselves.

**Comment by Angelodaniel** [comment]
@bcoe would having Insights work in dashboards have the possibility to split by tags as well? 
Like INP metrics per Custom tag?

**Comment by bcoe** [comment]
@Angelodaniel I might not be understanding, but can't you already do this today? If you're willing to create a custom chart?

<img width="2334" height="1328" alt="Image" src="https://github.com/user-attachments/assets/782a8d0b-2990-4035-9ff7-3ff12f4a188e" />

It will be possible to do the exact same on Insights charts out of the box 👍 , as they'll work identically.

**Comment by Angelodaniel** [comment]
I was just referencing to your update about having "insights" widgets. I don't know how different it is from what's currently already available hence my question. 

The goal I have is to view INP per (custom-tag) TvPlatform.
I guess with the new Global tags the Insights widgets should just work based on that so the answer is yes. But it wasn't released when I asked :) 

Still coming back on my initial point: 
This is how it currently looks in dashboards

<img width="902" height="269" alt="Image" src="https://github.com/user-attachments/assets/9de522bb-338d-4e26-80b5-a09eb7c71c62" />

This is what's possible in Trace explorer: 

<img width="1489" height="303" alt="Image" src="https://github.com/user-attachments/assets/484038d0-31a4-4e60-9f7d-8d661c1f0905" />
<img width="1480" height="279" alt="Image" src="https://github.com/user-attachments/assets/f55607f8-4858-4d28-a54d-d9ae282ce01f" />

Would really help if we can tweak the same time interval on Dashboard widgets regardless of data type
Why: With a more stable line it's easier to see trendlines going up or down over time. Otherwise the outliers mess up if it's going up or down.

**Comment by bcoe** [comment]
> Would really help if we can tweak the same time interval on Dashboard widgets regardless of data type

We have this near the top of the backlog for dashboards features we'd like to add 👍
